docker-compose.yml

```diff
@@ -0,0 +1,49 @@
+version: "3.4"
+
+x-logging:
```

Comment:
why not use this for the stac-db service too?

Reply:
@mishaschwartz can you clarify what you mean?